build/dependencies.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <PropertyGroup>
+    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
```

Comment:
1.6.1 shipped with .NET Core 1.1.0. Since we target LTS, we probably want 1.6.0 if we decide to explicitly reference `NETStandard.Library`.